desktop_version/src/FileSystemUtils.cpp

```diff
@@ -149,6 +151,40 @@ char *FILESYSTEM_getUserLevelDirectory()
 	return levelDir;
 }
 
+bool FILESYSTEM_directoryExists(const char *fname)
+{
+    return PHYSFS_exists(fname);
+}
+
+const char pathSeparator =
```

Comment:
This can be replaced with PHYSFS_getDirSeparator().

Reply:
Do you mean the entire FILESYSTEM_directoryExists() function? I'm a little confused here.

Reply:
No, you just use `PHYSFS_getDirSeparator()` instead of having to make your own `const char pathSeparator`.